### ISSUE TITLE
Dependencies

### ISSUE THREAD

**Issue by KarloDrachenauge** [cgtinker/BlendArMocap#53]
Hi, i get this as soon as i activate the addon. The strange thing is, that it already worked a while ago. but now it doesnt anymore:

Try to access dependencies
Dependency(module='mediapipe', package=None, name=None)
Exception in module register(): C:\Users\karls\AppData\Roaming\Blender Foundation\Blender\3.2\scripts\addons\BlendArMocap-release\__init__.py
Traceback (most recent call last):
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\scripts\modules\addon_utils.py", line 371, in enable
    mod.register()
  File "C:\Users\karls\AppData\Roaming\Blender Foundation\Blender\3.2\scripts\addons\BlendArMocap-release\__init__.py", line 45, in register
    ui_registration.register()
  File "C:\Users\karls\AppData\Roaming\Blender Foundation\Blender\3.2\scripts\addons\BlendArMocap-release\src\cgt_blender\interface\ui_registration.py", line 40, in register
    install_dependencies.import_module(module_name=dependency.module, global_name=dependency.name)
  File "C:\Users\karls\AppData\Roaming\Blender Foundation\Blender\3.2\scripts\addons\BlendArMocap-release\src\cgt_blender\utils\install_dependencies.py", line 54, in import_module
    globals()[global_name] = importlib.import_module(global_name)
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\site-packages\mediapipe\__init__.py", line 17, in <module>
    import mediapipe.python.solutions as solutions
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\site-packages\mediapipe\python\solutions\__init__.py", line 17, in <module>
    import mediapipe.python.solutions.drawing_styles
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\site-packages\mediapipe\python\solutions\drawing_styles.py", line 21, in <module>
    from mediapipe.python.solutions.hands import HandLandmark
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\site-packages\mediapipe\python\solutions\hands.py", line 39, in <module>
    from mediapipe.python.solution_base import SolutionBase
  File "C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\site-packages\mediapipe\python\solution_base.py", line 49, in <module>
    from mediapipe.modules.objectron.calculators import lift_2d_frame_annotation_to_3d_calculator_pb2
ImportError: cannot import name 'lift_2d_frame_annotation_to_3d_calculator_pb2' from 'mediapipe.modules.objectron.calculators' (C:\Users\karls\VHdZ\Progs\Blender\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\blender-3.2.0-alpha+master.4f2f3114b7be-windows.amd64-release\3.2\python\lib\site-packages\mediapipe\modules\objectron\calculators\__init__.py)

**Comment by cgtinker** [comment]
Not sure what causes this, which operating system, BlendArMocap- and Blender-version are you using?

Recently Google updated their buffer prototypes and mediapipe did not include the update in their dependencies. I don't know if that's connected in any way. However, it's probably worth trying the current development version.

https://github.com/cgtinker/BlendArMocap/tree/development

**Comment by bsenftner** [comment]
I've spent some time trying to see if installing the updated protocol buffers fixes things, but no luck so far.
Also tried simply running Blender's Python interactively and trying "import cv2" which is causing a dll load error. Tried copying opencv_world341.dll and opencv_world410.dll as mentioned on a github issue for mediapipe, but that does not seem to help. 

I've previously had OpenCV courses, so I'm attempting to see if installing a more complete OpenCV development environment inside Blender's python will grant me a progress brownie. I'll report back on progress.
